Packs/Troubleshoot/Scripts/TestInstance/TestInstance.yml

```diff
@@ -0,0 +1,43 @@
+args:
```

Comment:
this script should have `dependson` demisto rest api

Reply:
done